conn.go

```diff
@@ -964,6 +964,15 @@ func (c *Conn) executeQuery(qry *Query) *Iter {
 		}
 
 		return &Iter{err: x, framer: framer}
+
+	// for DowngradeConsistencyPolicy
+	case *RequestErrUnavailable:
```

Comment:
I dont think we need these cases. The err interface will still hold the underlying value as `*RequestErrUnavailable`